Detectors/FIT/FV0/simulation/include/FV0Simulation/Detector.h

```diff
@@ -122,14 +114,15 @@ class Detector : public o2::base::DetImpl<Detector>
 
   template <typename Det>
   friend class o2::base::DetImpl;
-  ClassDefOverride(Detector, 1);
+  ClassDefOverride(Detector, 1)
```

Comment:
please but back the ``;`` otherwise clang-format may apply funny alignment.

Reply:
Done